packages/swapper/src/swappers/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -291,13 +294,18 @@ export async function getTrade(
     )
   }
 
-  return Ok(promises.filter(isFulfilled).map(({ value }) => value))
+  return Ok(promises.filter(isFulfilled).map(({ value }) => value)) as unknown as Result<
```

Comment:
We shouldn't need to cast as `unknown`.

```suggestion
  return Ok(promises.filter(isFulfilled).map(({ value }) => value)) as Result<
```

Reply:
https://github.com/shapeshift/web/commit/214784c534228d7742d9626a363c6f8b038a664f